app.js

```diff
@@ -9,6 +10,10 @@ app.use((err, req, res, next) => {
     console.log(err);
 });
 
+app.get('/api', (req, res) => {
+    res.status(200).send(endpoints);
+});
```

Comment:
Nice, but please do create a controller for this. There is no need for a model since we are not querying the database, but a controller is still something I would recommend for this.